host/sgx/tests.c

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/host.h>
+#include <openenclave/internal/tests.h>
+#include "../hostthread.h"
+#include "sgxquoteprovider.h"
+
+static bool _has_quote_provider = false;
+
+static void _check_quote_provider(void)
+{
+    _has_quote_provider = oe_initialize_quote_provider() == OE_OK;
+}
+
+bool oe_has_quote_provider(void)
```

Comment:
Please add sgx into the function name
```suggestion
bool oe_has_sgx_quote_provider(void)
```

Reply:
Good point. Will do.